content/en/blog/_posts/2023-03-08-nodeport-dynamic-and-static-allocation.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: blog
+title: "Kubernetes 1.27: Avoid Collisions Assigning ports to Nodeport Services"
+date: 2023-03-08
+slug: nodeport-dynamic-and-static-allocation
+---
+
+**Author:** xuzhenglun
```

Comment:
Add your affiliation if possible.

Reply:
I need to confirm with the legal counsel whether it is possible to add